doc/usecases.md

```diff
@@ -0,0 +1,188 @@
+#Layer2
+
+Usage can refer to this [doc](layer2.md)
```

Comment:
[Layer 2 guide](layer2.md) demonstrates how to install Porter on Kubernetes with Layer 2 protocol.